main.html

```diff
@@ -12,9 +12,20 @@
 <body>
   <div class="container-fluid">
     <h1 id="name">Mindustry Mod Creator</h1><h6>By Paulao17</h6>
+    
```

Comment:
Why so many empty lines ?

Reply:
My bad. You can remove the empty lines of course.